docs/developer-file-overview.md

```diff
@@ -17,7 +17,8 @@ Zulip Terminal uses [Zulip's API](https://zulip.com/api/) to store and retrieve
 |                        |                     |                                                                                                        |
 | zulipterminal/cli      | run.py              | Marks the entry point into the application                                                             |
 |                        |                     |                                                                                                        |
-| zulipterminal/config   | color.py            | Color definitions or functions common across all themes                                                |
+| zulipterminal/config   | color.py            | Contains color definitions or functions common across all themes. For further details on themefiles    |
+|                        |                     | look at the theme contribution guide.                                                                  |
```

Comment:
The overview file doesn't need the extra line. For now I'd suggest simplifying your code and moving it to a comment in the file.